packages/react-components/stable-review/react-components.api.md

```diff
@@ -1035,6 +1036,12 @@ export interface TypingIndicatorStylesProps extends BaseCustomStyles {
     typingUserDisplayName?: IStyle;
 }
 
+// @internal
+export const _useContainerHeight: (containerRef: RefObject<HTMLElement>) => number | undefined;
+
+// @internal
+export const _useContainerWidth: (containerRef: RefObject<HTMLElement>) => number | undefined;
+
```

Comment:
Nice. Thanks for internally exporting these to avoid duplicates. Will update PR https://github.com/Azure/communication-ui-library/pull/1800 to use these.